styles/styles.css

```
@@ -1,7 +1,284 @@
 @import url("./colors.css");
 @import url("./fonts.css");
 
-.test {
-  color: var(--color-primary);
+/* ========================== HOME PAGE ================= */
+
+
+
+/* ==== MOBILE ==== */
+
+@media (max-width: 567px) {
+  .grid-container {
+    grid-template-columns: 1fr;
+  }
+
+  .poppins-style {
+    text-align: center;
+  }
+
+  .banner-style {
+    height: auto;
+    padding: 20px;
+  }
+
+  .home-page .image-container img {
+    width: 170px;
+    height: 150px;
+  }
+  
+  .home-page .text-img-container {
+    justify-content: center;
+    width: fit-content;
+    padding: 10px;
+  }
+
+  #title-banner {
+    font-size: 20px;
+    margin: 0 0 5px 0;
+  }
+
+  .icons-style {
+    gap: 10px;
+    margin: 0;
+  }
+  
+  .custom-icon {
+    width: 15px;
+    height: 15px;
+  }
```

quiza añadir .home-page adelante de algunas clases en el styles que pueden no estar del todo claras que peternecen a la parte del homepage, por ejemplo

```suggestion
.home-page .icons-style {
    gap: 10px;
    margin: 0;
  }

.home-page .custom-icon {
    width: 15px;
    height: 15px;
  }
```